test/docfx.Test/lib/YamlUtilityTest.cs

```diff
@@ -158,6 +158,37 @@ public void TestBoolean()
             Assert.Equal(new[] { true, true, true, false, false, false }, value2.Select(j => (bool)j).ToArray());
         }
 
+        [Fact]
+        public void TestNull()
```

Comment:
better to use `InlineData` to specify input and output? would be easier to add more test cases

Reply:
updated